go/buildhelper/helpertypes/types.go

```diff
@@ -0,0 +1,130 @@
+package helpertypes
+
+import (
+	"bytes"
+	"compress/gzip"
+	b64 "encoding/base64"
+	"io/ioutil"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/obscuronet/obscuro-playground/go/buildhelper/buildconstants"
+	"github.com/obscuronet/obscuro-playground/go/obscurocommon"
+)
+
+var IsRealEth bool
+
+func UnpackL1Tx(tx *types.Transaction) *obscurocommon.L1TxData {
+	if !IsRealEth {
```

Comment:
@otherview I really hate this. How hard would it be to shift our simulation code to creating txs that can be processed the same as "real" Eth transactions? Could we leave a TODO to do that, at a minimum? I worry this is a good route for bugs to creep in.

Reply:
Yeah.. I know what you mean!

I didn't want to change all the simulation txs as that would increase the scope of this work by quite a lot I reckon.
This is totally a not-so-clean way of doing it.

Adding a TODO - but we should definitely chat whether we want a routing approach (like this one) or change the simulation.